apps/client/src/pages/community/community-edit/community-edit.tsx

```diff
@@ -68,29 +59,81 @@ const CommunityEdit = () => {
     },
   });
 
-  const handlePutFeed = () => {
-    if (isDisabled || !category) {
-      return;
+  const { mutate: postImageUploadMutate } = useMutation({
+    ...MUTATION_QUERY_OPTIONS.POST_IMAGE(),
+  });
```

Comment:
```suggestion
  const { mutateAsync: postImageUploadMutate } = useMutation({
    ...MUTATION_QUERY_OPTIONS.POST_IMAGE(),
  });
```
이렇게 mutateAsync를 사용하면 new Promise를 생성하지 않아도 된다고 하네요!